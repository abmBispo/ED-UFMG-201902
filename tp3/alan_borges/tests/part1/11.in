128
letras inuteis e letras dispensaveis dizia ele que servico diverso prestam o d e o t tem quase o mesmo som o mesmo digo do b e do p o mesmo do s do c e do z o mesmo do k e do g etc sao trapalhices caligraficas veja os algarismos nao ha dous que facam o mesmo oficio e e e e admire a beleza com que um e um formam esta cousa que se exprime por agora dobre e tera multiplique por igual numero da e assim por diante mas onde a perfeicao e maior e no emprego do zero o valor do zero e em si mesmo nada mas o oficio deste sinal negativo e justamente aumentar um sozinho e um ponha lhe dous
q admire
q agora
q algarismos
q assim
q aumentar
q b
q beleza
q c
q caligraficas
q com
q cousa
q d
q da
q deste
q diante
q digo
q dispensaveis
q diverso
q dizia
q dobre
q ele
q em
q emprego
q esta
q etc
q exprime
q facam
q formam
q g
q ha
q igual
q inuteis
q justamente
q k
q lhe
q maior
q multiplique
q nada
q nao
q negativo
q no
q numero
q onde
q os
q p
q perfeicao
q ponha
q prestam
q quase
q s
q sao
q se
q servico
q si
q sinal
q som
q sozinho
q t
q tem
q tera
q trapalhices
q valor
q veja
q z
q a
q dous
q letras
q mas
q oficio
q zero
q por
q que
q um
q mesmo
q do
q o
q e
