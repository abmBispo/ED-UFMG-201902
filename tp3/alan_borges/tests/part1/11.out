1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
2
2
2
2
2
2
3
4
4
6
9
9
17
